src/main/java/club/sk1er/mods/autogg/tasks/data/Trigger.java

```diff
@@ -0,0 +1,32 @@
+package club.sk1er.mods.autogg.tasks.data;
+
+/**
+ * Data holder for a trigger in the server triggers JsonArray
+ *
+ * Trigger types:
+ * 0 = Normal
+ * 1 = Casual
+ * 2 = AntiGG
+ * 3 = AntiKarma
+ *
+ * @author ChachyDev
+ */
+
+public class Trigger {
+    private final int type;
```

Comment:
instead of "magic numbers", maybe make this an enum or something similar? that way there's a representation without needing to check the comments